docs/build/build_system.md

```diff
@@ -0,0 +1,71 @@
+# Build system
+
+The scp-machine build system is based on the principles of the [sc-machine](https://github.com/ostis-ai/sc-machine) build system. To learn more about them, read [sc-machine build system documentation](https://ostis-ai.github.io/sc-machine/build/build_system/).
+
+## Compilation steps
+
+This project uses CMake presets. These presets help manage different build configurations, making it easier to work with dependencies and streamline the build process.
+
+If you wish to use CLI instead, you can list all the available configurations in the `CMakePresets.json` file and pick one of the presets:
+
+```sh
+cmake --list-presets
```

Comment:
CMake Error: Could not read presets from /home/crafting-bench/work/conan-cars/scp-machine:
File not found: /home/crafting-bench/work/conan-cars/scp-machine/./ConanPresets.json

Reply:
You should run `conan export-pkg .` for built sc-machine, after then you should run `conan install .` for scp-machine